### PR TITLE
Add custom landing page documentation

### PR DESCRIPTION
**Pull request by urseberry**
### What does this PR do?
Add a new page to document how to set a custom landing page for an org. Add the page to the left menu under Org Settings.

### Motivation
https://datadoghq.atlassian.net/browse/DOCS-3328
Feature is going GA this week.

### Preview
https://docs-staging.datadoghq.com/uchen/custom-landing/account_management/org_settings/custom_landing/

### Additional Notes
A screen recording of setting and unsetting a custom landing page can be found on Slack in the #org-configs channel. Here is a direct link: https://dd.enterprise.slack.com/files/U02SY76473J/F03BT2EF5RT/screen_recording_2022-04-12_at_12.50.20_pm.mov

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by urseberry** [comment]
Ready for docs team review. Do not merge yet. Product will tell us when they release the feature and it is safe to merge.

File: content/en/account_management/org_settings/custom_landing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+kind: documentation
+---
+
+## Overview
+
+The Datadog organization landing page is the first page your users see when they log on to Datadog or navigate to the Datadog root page. Datadog sets a default landing page for your organization. If you use APM, Datadog sets the APM root as the landing page. If you don’t use APM, then the list of dashboards is the default landing view.
+
+As an alternative to the default page, Datadog allows administrators to set a dashboard as the landing page for the organization. A custom landing page helps a large or small organization control the narrative for their users.
+
+You can customize a dashboard with the information you want your users to see when they first log on to Datadog. Use Organization settings to set that dashboard as the custom landing page for your organization.
```
Comment (jtappa) [inline content/en/account_management/org_settings/custom_landing.md:12]:
```suggestion
You can customize a dashboard with the information you want your users to see when they first log on to Datadog. Use [Organization settings][1] to set that dashboard as the custom landing page for your organization.
```

**Review by jtappa — APPROVED** [review]
Small suggestion, but not a blocker, so I'm approving!

File: content/en/account_management/org_settings/custom_landing.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Custom Organization Landing Page
+kind: documentation
+---
+
+## Overview
+
+The Datadog organization landing page is the first page your users see when they log on to Datadog or navigate to the Datadog root page. Datadog sets a default landing page for your organization. If you use APM, Datadog sets the APM root as the landing page. If you don’t use APM, then the list of dashboards is the default landing view.
```
Comment (sseg-dd) [inline content/en/account_management/org_settings/custom_landing.md:8]: `list of dashboards is the default landing view.` inconsistent with "landing page" above?

Comment (urseberry) [reply content/en/account_management/org_settings/custom_landing.md]: I was trying to add some variety in wording. But after reading your comment, I see that the result is confusing. I changed that last "view" to be "page" like the other sentences.

**Pull request merged**